pdal/Options.cpp

```diff
@@ -50,7 +50,12 @@ std::string Option::toArg() const
 
 void Option::toMetadata(MetadataNode& parent) const
 {
-    parent.add(getName(), getValue());
+    // 'application' nodes on Stages are JSON
+    if (!Utils::iequals(getName(), "application"))
```

Comment:
Names are limited to lowercase (and in any case, aren't matched in a case sensitive way), so you can just do getName() == "application"